stix_shifter_modules/okta/stix_transmission/connector.py

```diff
@@ -116,24 +115,18 @@ def create_results_connection(self, query, offset, length, metadata=None):
             response_dict['message'] = f'Invalid metadata: {str(ex)}'
             ErrorResponder.fill_error(return_obj, response_dict, ['message'], connector=self.connector)
 
-        except ConnectionError:
-            response_dict['code'] = 300
-            response_dict['message'] = 'Invalid Host/Network Connection error'
-            ErrorResponder.fill_error(return_obj, response_dict, ['message'], connector=self.connector)
-        except RequestException:
-            if metadata:
-                return_obj = self.create_results_connection(query, offset, length, metadata)
-            else:
-                return_obj = self.create_results_connection(query, offset, length)
         except Exception as ex:
-            if "timeout_error" in str(ex):
-                response_dict['code'] = 300
-                response_dict['message'] = str(ex)
+            if "too_many_requests with max retry" in str(ex):
```

Comment:
I tested this part with 15 concurrent connections and this enters a dead cycle

Reply:
You should not retry to make a connection on 429 error. That part is implemented in the [RestApiClientAsync](https://github.com/opencybersecurityalliance/stix-shifter/blob/develop/stix_shifter_utils/stix_transmission/utils/RestApiClientAsync.py#L114) 
just make ErrorResponder.fill_error() instead

Reply:
We have updated changes as suggested for 429 response code.